plugins/base/proto/base.proto

```diff
@@ -20,8 +20,9 @@ service BasePlugin {
 // PluginType enumerates the type of plugins Nomad supports
 enum PluginType {
   UNKNOWN = 0;
-  DRIVER = 1;
-  DEVICE = 2;
+  BASE = 1;
```

Comment:
Why is `BASE` considered a plugin type?

Reply:
It shouldn't! Was being lazy for testing.